### PR TITLE
Add Access Token Custom Permission

### PR DESCRIPTION
**Pull request by ts-leojoseph-r**
# Description
Added only Access Token Custom Permission

## Links
[MINI-2678](https://jira.rakuten-it.com/jira/browse/MINI-2678)

# Checklist
- [x] I have read the [contributing guidelines](CONTRIBUTING.md).
- [x] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [x] I wrote/updated tests for new/changed code
- [x] I removed all sensitive data before every commit, including API endpoints and keys
- [x] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by corycaywood — COMMENTED** [review]
Looking good, but should we also add the permission check in `MiniAppScriptMessageHandler.fetchTokenDetails` in this PR?

File: CHANGELOG.md
Comment (ts-leojoseph-r) [reply CHANGELOG.md]: Oops. Updated now.

File: MiniApp/Classes/core/JavascriptBridge/MiniAppJavaScriptEnumerations.swift
Comment (ts-leojoseph-r) [reply MiniApp/Classes/core/JavascriptBridge/MiniAppJavaScriptEnumerations.swift:41]: Yeah, that's right. Added now.

File: MiniApp/Classes/core/JavascriptBridge/MiniAppJavaScriptEnumerations.swift
Comment (ts-leojoseph-r) [reply MiniApp/Classes/core/JavascriptBridge/MiniAppJavaScriptEnumerations.swift:35]: I feel it's good to have. User might use the quick actions to check the summary/ how-to-use in the description. Added them now.

**Review by corycaywood — APPROVED** [review]
Looks good 👍  But I think we shouldn't merge this to master yet because we don't want to release it in 3.1.0. Or maybe we could merge it to a feature branch? What do you think?

**Review by Climbatize — APPROVED** [review]
LGTM

**Comment by Climbatize** [comment]
> Looks good 👍 But I think we shouldn't merge this to master yet because we don't want to release it in 3.1.0. Or maybe we could merge it to a feature branch? What do you think?

I think that it's ok to merge it to master as long as we start the candidate branch on the commit before this merge and cheery pick what we are missing. That's now we can feel the lack of a develop branch ;)

**Comment by corycaywood** [comment]
> > Looks good 👍 But I think we shouldn't merge this to master yet because we don't want to release it in 3.1.0. Or maybe we could merge it to a feature branch? What do you think?
> 
> I think that it's ok to merge it to master as long as we start the candidate branch on the commit before this merge and cheery pick what we are missing. That's now we can feel the lack of a develop branch ;)

@Climbatize Yeah, I guess we could do that, but it sounds a little troublesome. i.e. there will probably be conflicts to resolve, and then it might be problematic when merging `candidate` back to `master` after the release.

Maybe another idea is that we could setup a CI job for feature branches?

**Pull request merged**
